### ISSUE TITLE
Issues with Sega bezels

### ISSUE THREAD

**Issue by biz101782** [thebezelproject/BezelProject#67]
Hey,. Love what y'all are doing!

Seeking some help....I am running retroarch with pico as the core.

I ran the bezel project on my genesis and 32x. The 32x is working, but the genesis isn't pickkmg up the bezels. (Also I did successfully install the Bezel project on mama).   Actually the sega / pico will pick up a handful of bezels ( right now I texted about 30, and it picked up 2)

Any advice on how to get the bezels on my sega roms?

**Comment by thebezelproject** [comment]
It sounds like you have a naming mismatch between your roms and the bezels. We use the no-intro naming scheme, so I would either suggest trying a no-intro rom set for genesis, or renaming your roms using a no-intro dat file.

**Comment by biz101782** [comment]
Thanks for the feedback.

Looking at my roms vs the PNG files.  And they seem accurate (no sure what u mean by no-intro rom)

For example:

My rom is
     X-men (USA)

The bezel PNG file is
    X-men (USA)


What exactly would I change on the rom?

**Comment by thebezelproject** [comment]
No-intro is a naming standard many rom releases use.

What core are you using for Mega Drive/Genesis? If you aren't using Genesis Plus GX, try that and see if there is any difference.

**Comment by biz101782** [comment]
Gotha.  Will do and will give it a shot.  Thanks for the advice!

**Comment by biz101782** [comment]
So I am now using gen plus gx.  So the bezels do work in restro arch.  But
when I load them in launchbox, the bezels arent coming thru.  Do you know
any settings in launchbox I need to adjust?  Thanks again!

**Comment by thebezelproject** [comment]
Did you change the core in selected in Launchbox to match gen plus gx also?

**Comment by biz101782** [comment]
Yep.  The core in launchbox is also gen plus gx. .  I plan on deleting the
roms from launchbox and re-importing to see if that does the trick.  It
maybe the way I imported.  Just the strange thing is there are 2-3 that do
work fine when loading from launchbox.

I have narrowed it down to is being a problem within launchbox....so let me
check that out (will not be til this upcoming weekend).

**Comment by thebezelproject** [comment]
Closing old issues. No follow-up.
